chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/Room.kt

```diff
@@ -15,35 +15,6 @@ data class Room(
         val lastMessageAt: String?,
         val createdAt: String,
         var updatedAt: String,
-        var deletedAt: String
-) {
-    @SerializedName("member_user_ids")
-    private var _memberUserIds: MutableSet<String>? = null
-    val memberUserIds: Set<String>
-        get() = memberUserIds()
-
-    private fun memberUserIds(): MutableSet<String> = _memberUserIds
-            ?: mutableSetOf<String>().also { _memberUserIds = it }
-
-    internal fun removeUser(userId: String) {
-        memberUserIds() -= userId
-    }
-
-    internal fun addUser(userId: String) {
-        memberUserIds() += userId
-    }
-
-    internal fun addAllUsers(userIds: Set<String>) {
-        memberUserIds().addAll(userIds)
-    }
-
-    override fun equals(other: Any?) = (other is Room) && other.id == this.id
-
-    override fun hashCode(): Int { return id.hashCode() }
-
-    fun deepEquals(room: Room) =
-            room.name == this.name &&
-                    room.pushNotificationTitleOverride == this.pushNotificationTitleOverride &&
-                    room.customData == this.customData &&
-                    room.isPrivate == this.isPrivate
-}
+        var deletedAt: String?,
+        val memberUserIds: Set<String>
```

Comment:
Let's call this out (fix spooky effect at a distance, with potential data races) in the changelog.

Reply:
Created [PR](https://github.com/pusher/chatkit-android/pull/167) addressing that.